compiler/rustc_mir_dataflow/src/framework/graphviz.rs

```diff
@@ -206,12 +204,7 @@ where
     A: Analysis<'tcx>,
 {
     body: &'mir Body<'tcx>,
-    // The `RefCell` is used because `<Formatter as Labeller>::node_label`
-    // takes `&self`, but it needs to modify the analysis. This is also the
-    // reason for the `Formatter`/`BlockFormatter` split; `BlockFormatter` has
-    // the operations that involve the mutation, i.e. within the `borrow_mut`.
```

Comment:
Does this `Formatter`/`BlockFormatter` split still exist? Should it be removed?

Reply:
I think the comment was overstated. The RefCell was part of the reason for the separation, but not the only reason. `Formatter` and `BlockFormatter` both have numerous methods, and multiple `BlockFormatters` can exist for a single `Formatter`. I'm not even sure if they could be sensibly combined, and keeping them separate seems reasonable.